src/main/java/core/basesyntax/constructors/Colors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.constructors;
+
+public enum Colors {

```

Comment:
Also, enums in Java should be singular.

Reply:
Singular, you mean to go in one line, one after another? I checked at StackOverflow and seems it is :)

Reply:
now its clear about forms :D